src/navigation/stacks/TransactionStackNavigator.tsx

```diff
@@ -0,0 +1,22 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import TransactionsScreen from '../../screens/Transactions/TransactionsScreen';
+
+const TransactionStack = createNativeStackNavigator();
+
+/**
+ * Stack navigator for the transaction/invoice screens.
+ */
+export default function TransactionStackNavigator() {
+  return (
+    <TransactionStack.Navigator
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <TransactionStack.Screen
+        name="TransactionsScreen"
```

Comment:
Renamed this from "Transactions" to "TransactionsScreen" – screen name had a duplicate in BottomNav.